homeassistant/components/apple_tv/__init__.py

```diff
@@ -130,7 +130,7 @@ def request_configuration(hass, config, atv, credentials):
         devices = ["No device(s) found"]
 
     hass.components.persistent_notification.async_create(
-        "The following devices were found:<br /><br />" + "<br /><br />".join(devices),
+        f"The following devices were found:<br /><br />{'<br /><br />'.join(devices)}",
```

Comment:
IMHO, this makes it really hard to read.
```suggestion
        list_devices = '<br /><br />'.join(devices)
        f"The following devices were found:<br /><br />{list_devices}",
```

Reply:
Good idea!